client/lib/products-values/index.js

```diff
@@ -173,6 +175,20 @@ export function isMonthly( product ) {
 	return parseInt( product.bill_period, 10 ) === PLAN_MONTHLY_PERIOD;
 }
 
+export function isYearly( product ) {
+	product = formatProduct( product );
```

Comment:
can we prefer immutability over mutating the arguments here for the sake of keeping down the complexity?

```js
export function isYearly( product ) {
	const formattedProduct = formatProduct( product );

	assertValidProduce( formattedProduct );

	return parseInt( product.bill_period, 10 ) === PLAN_ANNUAL_PERIOD;
}
```

alternatively we can rename the arg…

```js
export function isYearly( rawProduct ) {
	const product = formatProduce( rawProduct );

	…
}
```